### ISSUE TITLE
Ability to pass go structure to update query

### ISSUE THREAD

**Issue by TomasWeg** [genjidb/genji#404]
Add the ability to pass a go structure as argument while updating an entry, like while inserting.
```go
database.Exec("UPDATE test SET ? WHERE id = 123", &data)
```

**Comment by cbrake** [comment]
Neat idea.

We had a discussion on slack that might be related to this:

![image](https://user-images.githubusercontent.com/402813/119358075-41246880-bc76-11eb-9bf4-5dc898ef5800.png)

I'm not very experienced with databases/SQL, so not sure if REPLACE or being able to pass in an entire structure makes the most sense.

**Comment by asdine** [comment]
Next release will contain a new feature that will allow doing that in one query

```sql
INSERT INTO test VALUES ? ON CONFLICT DO REPLACE
```

Fixed by 7757022ea604f3682fabee0b0b92a6b2d0dc049f and 357adc9f31fd2f1035ca48687cfc5986f061f9f9.
